client/src/features/transactions/SearchButton.css

```
@@ -0,0 +1,83 @@
+
+.search-popover-title {
+  margin: 0;
+}
+
+.search-popover-input {
+  padding: var(--spacing-xs) var(--spacing-s);
+  border: 1px solid var(--color-light);
+  border-radius: var(--spacing-xs);
+  background: var(--color-background);
+  color: var(--color-white);
+  font: 400 16px/1.5 'Quicksand', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Quicksand" (font-family-name-quotes)

(font-family-name-quotes)